dev/preview/BUILD.yaml

```diff
@@ -8,18 +8,23 @@ packages:
       commands:
         - ["echo", "done"]
 scripts:
+  - name: configure-gcp-access
+    description: Configures the workspace so that it has access to the relevant GCP resources.
+    script: ./workflow/preview/configure-gcp-access.sh
+
+  - name: configure-workspace
+    description: Configures the workspace so that it has access to development resources (dev, harvester) as well as your preview environment.
+    script: ./workflow/preview/configure-workspace.sh
+
   - name: build
     description: Build all packages needed to deploy Gitpod to preview environments
     script: ./workflow/preview/build.sh
 
-  - name: get-credentials
```

Comment:
This is not currently used and the description is wrong, so decided just to get rid of it ☺️